lua/spider.lua

```diff
@@ -147,6 +147,14 @@ function M.motion(key)
 	if isOperatorPending and key == "e" then col = col + 1 end
 
 	vim.api.nvim_win_set_cursor(0, { row, col })
+
+-- set plug keymaps
```

Comment:
even with the `end` added, the plugin still won't run, since you set the `<Plug>` keymaps inside the function they are supposed to call. You would need set them up elsewhere, e.g. in the setup function (which is then becoming mandatory for dot repeats)